packages/core/src/storage.ts

```diff
@@ -1,41 +1,24 @@
 //
-// Copyright © 2020 Anticrm Platform Contributors.
-// 
+// Copyright © 2021 Anticrm Platform Contributors.
+//
 // Licensed under the Eclipse Public License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License. You may
 // obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
-// 
+//
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// 
+//
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
 
-import type { Emb, Doc, Ref, Class, Data, Obj } from './classes'
-import type { Tx, TxCreateObject, TxAddCollection } from './tx'
-
-import core from './component'
+import type { Class, Data, Doc, Ref } from './classes'
+import type { Tx } from './tx'
 
 export type DocumentQuery<T extends Doc> = Partial<Data<T>>
 
 export interface Storage {
-  findAll<T extends Doc>(_class: Ref<Class<T>>, query: DocumentQuery<T>): Promise<T[]>
-  tx(tx: Tx): Promise<void>
-}
-
-export class TxProcessor {
-  tx (tx: Tx): Promise<void> {
-    switch(tx._class) {
-      case core.class.TxCreateObject:
-        return this.txCreateObject(tx as TxCreateObject<Doc>)
-      case core.class.TxAddCollection:
-        return this.txAddCollection(tx as TxAddCollection<Emb>)
-    }
-    return Promise.resolve()
-  }
-
-  protected async txCreateObject (tx: TxCreateObject<Doc>): Promise<void> {}
-  protected async txAddCollection (tx: TxAddCollection<Emb>): Promise<void> {}
+  findAll: <T extends Doc>(_class: Ref<Class<T>>, query: DocumentQuery<T>) => Promise<T[]>
+  tx: (tx: Tx) => Promise<void>
```

Comment:
why `f(): x` replaced with `f: () => x`?

Reply:
Because of rule: 
https://github.com/typescript-eslint/typescript-eslint/blob/v4.23.0/packages/eslint-plugin/docs/rules/method-signature-style.md

ts-standard 10.0 had same output for me as eslint.

<img width="1478" alt="Screenshot 2021-06-06 at 15 27 48" src="https://user-images.githubusercontent.com/477235/120917871-c29ae280-c6db-11eb-999c-6d4591939088.png">